privacy_evaluator/output/user_output.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pandas as pd
+from matplotlib import pyplot as plt
+
+from typing import Tuple
+
+
+class UserOutput:
+
+    """User Output Class"""
+
+    def __init__(
+        self,
+        attack_data_y: np.ndarray,
+        privacy_risk: np.ndarray,
+        all_labels: np.ndarray,
+    ):
+        """
+        Initilaizes the Class with values
+        """
+        self.attack_data_y = attack_data_y
+        self.privacy_risk = privacy_risk
+        self.all_labels = all_labels
+
+    def histogram_top_k(
+        self, k: int = 10, show_diagram: bool = True
+    ) -> Tuple[np.ndarray, np.ndarray]:
+        """
+        Draws a historgam of the top k points with biggest Privacy score
```

Comment:
reformulate to something like: "Draw histogram of class distribution of the k points with highest privacy risk score."

Reply:
done